packages/fauna/package.json

```diff
@@ -21,10 +21,19 @@
     "access": "public"
   },
   "scripts": {
-    "test": "exit 0"
+    "reset": "export FAUNA_ADMIN_KEY=secret;export FAUNADB_DOMAIN=localhost; export FAUNADB_SCHEME=http;export FAUNADB_PORT=8443;fauna-schema-migrate apply all",
```

Comment:
This can be cleaned up a bit and make it work in all types of system using cross-env

For reference:
https://github.com/pbteja1998/nextjs-starter/blob/main/package.json#L20

Reply:
Thanks, good idea!